blog-posts/emails/article.md

```diff
@@ -0,0 +1,329 @@
+---
+published: false
+title: 'From Zero to Hero... Send AWS Ses Emails Like a Pro!'
+cover_image: https://raw.githubusercontent.com/pchol22/kumo-articles/master/blog-posts/emails/assets/cover.png
+tags: serverless, AWS, SES, Emails
+series:
+canonical_url:
+---
+
+_This article assumes the reader has **really basic** knowledge of SES, like being able to send simple emails using SES and Lambda or to verify an identity._
+
+As you may already know, AWS SES offers a great way to send emails with your AWS serverless app. Coupled with the AWS SDK and Lambda functions, it allows you to programmatically send emails to your users with minimal setup. However, this ease of use comes at the price of potential mistakes and pitfalls that can happen at every step of your coding journey. Let me guide you through **three** of these issues and help you design beautiful emails that never end up in your users spam.
+
+## 🚧 🚧 🚧 Three obstacles on your road to clean email communication
+
+A few months ago, I went through the implementation of transactional emails sending on the project I was working on, and identified three major pitfalls that every developer should be aware of.
+
+Using only **minimal configuration** to send your emails:
+
+- 🚧 You will be prevented from sending messages if your sender reputation goes down.
+- 🚧 Your emails will probably end up in some of your users spams.
+- 🚧 Your emails will probably look bad on some clients (⚠️ gmail is one of them!)
```

Comment:
Look bad, or won't display at all on some clients that do not support html (sometimes for security reasons, companies forbid html and only allow plain text)